src/main/java/org/apache/pulsar/manager/service/impl/BrokerStatsServiceImpl.java

```diff
@@ -131,7 +131,12 @@ private void scheduleCollectStats() {
             clusterLists.forEach((clusterMap) -> {
                 String cluster = (String) clusterMap.get("cluster");
                 Pair<String, String> envCluster = Pair.of(env.getName(), cluster);
-                collectStatsServiceUrls.put(envCluster, (String) clusterMap.get("serviceUrl"));
+                String webServiceUrl = (String) clusterMap.get("serviceUrl");
+                if (webServiceUrl.contains(",")) {
+                    String[] webServiceUrlList = webServiceUrl.split(",");
+                    webServiceUrl = webServiceUrlList[0];
```

Comment:
here you are keeping only the first one.
what about if it is down ?

Reply:
Thanks, @eolivelli, I fixed it, PTAL